### DISCUSSION TITLE
247

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#732] [Comments]
http://jojoldu.tistory.com/247

**Comment by jojoldu** [comment]
작성자: joshua.qa

앗 오셨었군요!! 마지막 사진에 제가 보이네요~ 정말 좋은 세미나였습니당

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

엇? 종립님과 세미나장 가면서 오늘 못오신다고 얘기를 들어서 못오셨는줄알았습니다 ㅠ
오신줄알았으면 인사라도 드리는건데 ㅠ

**Comment by jojoldu** [comment]
작성자: soulduse

항상 좋은글 공유해주셔서 감사드립니다~

정말 강연듣는것 처럼 잘 정리해놓으셨네요

자극 받고갑니다 :)

**Comment by jojoldu** [comment]
작성자: JamesGold

이 티스토리를 구독해야 할 이유가 생겼네요

감사합니다.

**Comment by jojoldu** [comment]
작성자: seungdols

이분 레전더리셔서 가고 싶었는데 요즘 바빠서 못갔네요 ㅠ 
좋은 후기 거기에 자세한 요약까지 항상 감사합니다.

**Comment by jojoldu** [comment]
작성자: 이동욱

좋은글 감사합니다.

**Comment by jojoldu** [comment]
작성자: 심병기

정말 좋은 내용 공유해주셔서 감사합니다

**Comment by jun0517** [comment]
정말 좋은 글 감사합니다.
존경스럽습니다.
